tests/cql/CqlDateTimeOperatorsTest.xml

```diff
@@ -470,51 +471,51 @@
 	<group name="From Github issue #29">
 		<test name="DateTimeA">
 			<expression>@2017-03-12T01:00:00-07:00</expression>
-			<output>@2017-03-12T01:00:00</output>
+			<output>@2017-03-12T01:00:00-07:00</output>
```

Comment:
I don't understand the purpose of this test or the similar ones that follow.  I especially don't understand why the expected output was to _drop_ the timezone offset.  That can't be right.